packages/qwik/src/core/component/component.unit.tsx

```diff
@@ -157,50 +157,41 @@ describe('q-component', () => {
             value="1"
           />
           <Input2 value="2" />
-          <Input5 value="5" type="text" />
-          <Input6 value="6" type="number" />
+          <Input3 value="3" type="text" />
+          <Input4 value="4" type="number" />
         </>
       );
     });
   });
 
   test('custom function types should work', () => () => {
     type TestProps = PropsOf<'h1'> & {
-      plain$?: () => void;
       qrl$?: QRL<() => void>;
     };
-    const Test1 = component$<TestProps>(({ plain$, qrl$, ...props }) => {
```

Comment:
Well, this is a test, even though it's not recommended we should still check that it works this way, no?

How about adding a comment above the plain$ definition saying you shouldn't do it but it works?

Reply:
Hmmm so it's only the eslint plugin that complains but it works? Can't it lead to buggy behaviors? I think it shouldn't be in the test if the eslint plugin says it shouldn't be used like that. Or the eslint plugin needs to be fixed.

Reply:
fair enough